asn1js/index.d.ts

```diff
@@ -0,0 +1,672 @@
+// Type definitions for asn1js
+// Project: https://github.com/PeculiarVentures/ASN1.js
+// Definitions by: Stepan Miroshin <https://github.com/microshine>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+declare module "asn1js" {
```

Comment:
could you strip `declare module` ?
https://github.com/Microsoft/types-publisher#fields-in-datadefinitionsjson
> ProperModule: Only top-level import and export declarations. Preferred